tests/unit/moduleapi/mallocsize.tcl

```diff
@@ -0,0 +1,11 @@
+set testmodule [file normalize tests/modules/mallocsize.so]
+
+
+start_server {tags {"modules"}} {
+    r module load $testmodule
+
+    test {MallocSize of raw, string and dict} {
+        assert_equal [r mallocsize.set key 12 abc k1 v1 k2 v2] {OK}
+        assert_morethan [memory_usage key] 100
+    }
```

Comment:
not sure how you got to `100`
i'm worried that if one of the primitive types will be completely broken (always returning 0), we won't spot it.
we can go around it is by repeating this command several times, each type testing just part of the data structure (e.g. skipping the dict).
WDYT?